packaging/rpm/build-package.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+set -eux  # Exit on error, print commands
+
+RPM_BUILD_ROOT=/workspace/build
+
+mkdir -p "$RPM_BUILD_ROOT"
+cd "$RPM_BUILD_ROOT"
+
+mkdir -p BUILD BUILDROOT RPMS SOURCES SPECS SRPMS
+cp /workspace/ice/packaging/rpm/ice.spec "$RPM_BUILD_ROOT/SPECS"
+
+cd "$RPM_BUILD_ROOT/SOURCES"
+spectool -g "$RPM_BUILD_ROOT/SPECS/ice.spec"
+
+dnf builddep --spec "$RPM_BUILD_ROOT/SPECS/ice.spec" -y --setopt=multilib_policy=all
```

Comment:
This command installs the build dependencies, and we should probably move this to the Dockerfile to avoid installing then with each build.

Currently we build the container in each run, but we want to move the container build to its own job.